### ISSUE TITLE
Invalid HTML - nested <a> inside <button>

### ISSUE THREAD

**Issue by mirekys** [VSCHT/restaurovani#28]
There are mulltiple occurences of nesting links & buttons inside each other, like this:
https://github.com/VSCHT/restaurovani/blob/d41c84fe42f81a9206b761bf47a1e08caf85ca6d/sites/restaurovani-site/site/restaurovani_site/templates/semantic-ui/restaurovani_site/base_page.html#L136C17-L142C26

Nesting `<a>` links inside `<button>` element and vice versa violates HTML spec and produces functional & accessibility issues see here: https://giovanicamara.com/blog/nested-buttons-and-links/, if an `<a>` element needs to look like a button, that should by achieved by CSS only
